polaris_wyre/wyre/api.py

```diff
@@ -0,0 +1,68 @@
+from urllib.parse import urljoin
+
+import requests
+
+from polaris_wyre.helpers.exceptions import WyreAPIError
+from .dtos import TransferData
+
+TEST_BASE_URL = "https://api.testwyre.com"
```

Comment:
Wouldn't be good to set this address in the settings or even as an env var and import it here?

Reply:
Not this case, we'll receive the `api_url` as a dependency. So I use this test URL as the default value for it.

Reply:
But wouldn't be easier to maintain keeping int in an env var or in the settings to set this default value? I mean, to have the external URLs concentrated in just one file.

Reply:
As we talked private, we'll let this way.